schema/to31.xsl

```diff
@@ -59,6 +59,10 @@
 
   <!-- Additions in attributes.mod -->
 
+  <!-- Remove new MusicXML 3.1 elements -->
```

Comment:
Should this be "Remove new MusicXML 4.0 elements?"

Reply:
@mscuthbert Yes indeed. Thanks for catching this! I have fixed it here and two other places in this file.